_inc/client/components/admin-notices/index.jsx

```diff
@@ -7,11 +7,16 @@ const AdminNotices = React.createClass( {
 		let $vpNotice = jQuery( '.vp-notice' );
 		if ( $vpNotice.length > 0 ) {
 			$vpNotice.each( function () {
+				if ( jQuery( this ).children( '.vp-message' ).children( 'p' ).html().includes( 'VP08' ) ) {
+					jQuery( this ).children( '.vp-message' ).children( 'p' ).html( 'Your VaultPress subscription is no longer active. Please confirm the status of your account. <a href="https://dashboard.vaultpress.com/account/">View account</a>' );
```

Comment:
The problem I see with that is the original string is a VaultPress notice that works correctly in the context of the VaultPress plugin. If we change it there, it will alter the valid notice for users who are using VaultPress without Jetpack, which is also not a great solution. I'm all ears for solutions, but I'm not sure fixing it in VaultPress is the right one.

With that said, you're right, this should be translated. Perhaps we should just translate this particular string in Jetpack?

Another solution might be to leave the message alone and simply strip out the second link so the notice is not malformed. Thoughts?